CoreCompatibility/Core_1_20_R1/src/main/java/me/deecaad/core/compatibility/nbt/NBT_1_20_R1.java

```diff
@@ -40,12 +40,12 @@ public void copyTagsFromTo(@NotNull ItemStack fromItem, @NotNull ItemStack toIte
 
         if (path == null) {
             nms.setTag(from.copy());
-            toItem.setItemMeta(nms.getBukkitStack().getItemMeta());
+            toItem.setItemMeta(getBukkitStack(nms).getItemMeta());
```

Comment:
This is interesting... I wonder how that can ever be null... Especially why it would only be present in 1.20. This definitely should be looked at soon.